pkg/crc/cluster/csr.go

```diff
@@ -41,12 +42,16 @@ func approveNodeCSR(ocConfig oc.Config, signerName string) error {
 		if len(csr.Status.Conditions) != 0 {
 			continue
 		}
-		if signerName != csr.Spec.SignerName {
-			logging.Debugf("Unexpected unapproved csr %s (signerName: %s)", csr.Metadata.Name, csr.Spec.SignerName)
+		var signerName string
+		if csr.Spec.SignerName != nil {
+			signerName = *csr.Spec.SignerName
```

Comment:
@cfergeau you mean here?

Reply:
yep, in that block. I thought I had pushed a fix for this already